src/maps/process.js

```diff
@@ -200,6 +200,43 @@ async function counties(level = 'county') {
   return geo;
 }
 
+async function hrr(level = 'hrr') {
+  const geo = require(`./raw/hrr/hrr.reprojected.geo.json`);
+
+  const infos = geo.features
+    .map((feature) => {
+      const id = String(feature.properties.hrr_num);
+      // don't know why but there is this frame
+      feature.geometry.coordinates = feature.geometry.coordinates.filter((d) => {
+        return !(d[0][0][0] === -26.069579678452456 && d[0][0][1] === 13.509452429458069);
```

Comment:
Can you define these magic numbers in a comment or pull them out into named variables?